development.mediawiki

```diff
@@ -267,5 +267,5 @@ Anyway, this is a workaround valid only in the case a tag synonym cannot be crea
 
 '''New releases''' of your components '''SHOULD''' be announced through different channels: FIWARE Blog, dedicated Twitter or similar channels, your own Blog ... 
 
-Support '''responsiveness''' is a '''MUST'''. If a feature is not implemented / not available, an alternative '''workaround''' '''MUST''' be provided.
+Support '''responsiveness''' is a '''MUST''' in case your enabler has an Web interface. If a feature is not implemented / not available, an alternative '''workaround''' '''MUST''' be provided.
```

Comment:
APIs cannot be responsive right?

Reply:
I was not referring to Responsive Design but to respond quickly to support issues :)

Reply:
I think this was not clear, I think we should anyhow recall that support to the community, unless funded, is best effort. If a feature is not supported, is not supported, there may be not work around :)

Reply:
Agree - the  _responsive_  text as written could be clearer.  My suggestion would be something like

>To build and maintain the FIWARE community, it is essential that the team responsible 
> for each product to openly engage with the wider development community. 
>\- therefore responding to User Support requests in a timely manner is a '''MUST'''.
>*  If an external developer has found an issue with existing functionality which cannot be fixed in a timely manner, a known issue '''MUST''' be added to the documentation.
>* If an external developer has found a fixable issue with existing functionality an alternative '''workaround''' '''MUST''' be provided, and the code patched
>*  If an external developer suggests  support for a new feature is not implemented / not available, an alternative '''workaround''' '''SHOULD''' be provided where possible.

This then nicely flows on to the next line.

>Afterwards it can be decided whether it is necessary / worth adding such a new feature to 
>your backlog. 

There is a difference here between someone exposing a bug in existing agreed functionality ( bug fixing really should be a **must**) and someone misinterpreting the existing functionality and saying _Why doesn't it do X?_ - the latter can be added to the backlog if it is seen as useful.

Where there is insufficient time/resources to fix bugs, adding a known issue to the documentation is a minimal solution showing respect to the community _Yes we know it doesn't it do X ... yet_

Reply:
@chicco785 would you be able to update this PR so that the content suggested by @jason-fox is added?

Reply:
Not sure it is the right place, but we should incentive the community to contribute. My feeling is still that most people engaging with FIWARE expect someone in FIWARE solve the issue, while some time things are really trivial.